Library/Formula/ice.rb

```diff
@@ -8,7 +8,11 @@ class Ice < Formula
   option 'doc', 'Install documentation'
   option 'demo', 'Build demos'
 
-  depends_on 'berkeley-db'
+  resource "berkeley-db" do
+    url 'http://download.oracle.com/berkeley-db/db-5.3.28.tar.gz'
+    sha1 'fa3f8a41ad5101f43d08bc0efb6241c9b6fc1ae9'
+  end
+
   depends_on 'mcpp'
   depends_on :python => :optional
 
```

Comment:
I think I need to move the revision clause from the internal Berkeley stage to the ice configuration as I suppose you want to force the update even for those who have the same version currently installed 

Cyril Scetbon

> On 22 Feb 2015, at 02:56, Dominyk Tiller <notifications@github.com> wrote:
> 
> In Library/Formula/ice.rb:
> 
> > @@ -8,7 +8,11 @@ class Ice < Formula
> >    option 'doc', 'Install documentation'
> >    option 'demo', 'Build demos'
> >  
> > -  depends_on 'berkeley-db'
> > +  resource "berkeley-db" do
> > +    url 'http://download.oracle.com/berkeley-db/db-5.3.28.tar.gz'
> > +    sha1 'fa3f8a41ad5101f43d08bc0efb6241c9b6fc1ae9'
> > +  end
> > +
> >    depends_on 'mcpp'
> >    depends_on :python => :optional
> >  
> can you stick a depends_on "openssl" here, and then a revision 1 underneath the sha1 above? Thanks!
> 
> —
> Reply to this email directly or view it on GitHub.
>